packages/consumption/src/modules/attributes/AttributesController.ts

```
@@ -355,6 +363,14 @@ export class AttributesController extends ConsumptionBaseController {
     }
 
     public async createSharedLocalAttribute(params: ICreateSharedLocalAttributeParams): Promise<LocalAttribute> {
+        const parsedParams = CreateSharedLocalAttributeParams.from(params);
+        if ("tags" in parsedParams.content && parsedParams.content.tags) {
+            const tagValidationResult = await this.validateTags(parsedParams.content.tags, parsedParams.content.toJSON().value["@type"]);
+            if (tagValidationResult.isError()) {
+                throw tagValidationResult.error;
+            }
+        }
+
```

I'm not sure if we want to have the validation here in general. This would be executed if a peer shared IdentityAttribute is created. Do we actually want to throw an error, if a peer shares an Attribute with an invalid tag or do we only want to validate creating an own RepositoryAttribute. @jkoenig134 Wdym? 

This might also have implications on the place of the succession validation. Furthermore, it must be handled analogously in the RequestItemProcessors.

---

yep